src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -18,6 +19,17 @@ export async function updateUserActiveEnterprise({ enterpriseCustomer }) {
   return getAuthenticatedHttpClient().post(url, formData);
 }
 
+/**
+ * TODO
+ */
```

Comment:
nit: todo

Reply:
[inform] We've been deferring on docstrings on PRs to the feature branch for once the route loaders migration is a bit further along with the UI layer integration. Will be cleaned up then! 😄